packages/appsync-emulator-serverless/lambdaSource.js

```diff
@@ -54,6 +60,21 @@ const lambdaSource = async (
       handlerMethod,
       payload,
     });
+  } else if (fnConfig.runtime && fnConfig.runtime.indexOf('go') >= 0) {
+    child = fork(GoRunner, [], {
+      env: {
+        ...process.env,
+        ...dynamodbTableAliases,
+        DYNAMODB_ENDPOINT: dynamodbEndpoint,
+      },
+      stdio: [0, 1, 2, 'ipc'],
+    });
+
+    child.send({
+      serverlessDirectory,
+      handlerMethod: goHandlerMethod,
+      payload,
+    });
```

Comment:
this interface is exactly the same as python except for the runner, we may as well handle it in the same block

(also, let's use includes)

```
if(fnConfig.runtime && !fnConfig.runtime.includes('node'))
  const runner = determineRunner(fnConfig.runtime)
  ...
}
```